components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -298,6 +298,11 @@ abstract class EngineSession(
              */
             SCRIPTS_AND_SUB_RESOURCES(1 shl 31),
 
+            /**
+             * Indicates that content that would have been blocked has instead been replaced with a shim.
+             */
+            SHIMMED(0),
```

Comment:
I think we should update the comment here to indicate that `SHIMMED` is only used to read and there's no need to actively configure it via `select`, or otherwise.

Since we use the same value as NONE, configuring it with select wouldn't make sense, but it's still possible using our current API i.e. you could write `TrackingProtectionPolicy.select(arrayOf(TrackingCategory.SHIMMED)` which we wouldn't want to do?

Reply:
That's right, we only want to use it for categorization propose. I will add the comment thanks :)

Reply:
Updated :)